ultralytics/engine/exporter.py

```diff
@@ -890,6 +890,8 @@ def export_saved_model(self, prefix=colorstr("TensorFlow SavedModel:")):
             if self.args.data:
                 f.mkdir()
                 images = [batch["img"].permute(0, 2, 3, 1) for batch in self.get_int8_calibration_dataloader(prefix)]
+                if len(set(self.imgsz)) == 2:
+                    images = [img[:, : self.imgsz[0], : self.imgsz[1], :] for img in images]
                 images = torch.cat(images, 0).float()
```

Comment:
@ambitious-octopus @glenn-jocher I tested directly resize the image also works! (using `full_integer_quant` model instead of the `yolo11n_int8` model)
we can simply update these two lines to:
```python
images = [batch["img"] for batch in self.get_int8_calibration_dataloader(prefix)]
images = torch.nn.functional.interpolate(torch.cat(images, 0).float(), size=self.imgsz).permute(0, 2, 3, 1)
```

Reply:
@Laughing-q got it, thank you!

@ambitious-octopus please update this PR to just LQ's proposed changes here.

Reply:
@ambitious-octopus Yes! I think we are good to go with the changes I proposed as we last week tested this way wouldn't affect precision that much to the final calibrated model, and it's minimal twos lines change.